src/linear_algebra/clifford_algebra/equivs.lean

```diff
@@ -89,3 +89,104 @@ alg_equiv.of_alg_hom to_complex
 attribute [protected] Q
 
 end clifford_algebra_complex
+
+/-! ### The clifford algebra isomorphic to the quaternions -/
+namespace clifford_algebra_quaternion
+
+open_locale quaternion
+open quaternion_algebra
+
+variables {R : Type*} [comm_ring R] (c₁ c₂ : R)
+
+/-- The quadratic form. -/
```

Comment:
This docstring isn't very descriptive if you don't know the entire story already.